libcontainer/cgroups/systemd/common.go

```diff
@@ -518,3 +522,19 @@ func addCpuset(cm *dbusConnManager, props *[]systemdDbus.Property, cpus, mems st
 	}
 	return nil
 }
+
+func getUnitType(unitName string) string {
```

Comment:
Placeholder. Can probably do something more smooth. This is just so we can start discussing the best way to solve it.

Reply:
We can probably just use `strings.ToTitle()` here.

Reply:
Yeah, was thinking about that. Or should we have an allowlist like this, to ensure we don't forward "bad" strings? Doesn't matter that much to me

Reply:
```go
validStuff := map[string]bool {
     "Slice": true
     "Scope": true
 }
  
 if validStuff[strings.ToTitle(type)] {
     return type
 }
 return err
```

Can probably do something like this to validate it.

Reply:
We can either have `.slice` or `.scope` (see `getUnitName()`).

I suggest we add `getUnitType` which is very similar to `getUnitName`.